mirrord-agent/src/main.rs

```diff
@@ -294,6 +317,11 @@ impl ClientConnectionHandler {
             LayerTcp::PortUnsubscribe(port) => self.tcp_sniffer_api.port_unsubscribe(port).await,
         }
     }
+
+    async fn handle_client_steal_tcp(&mut self, message: LayerStealTcp) -> Result<(), AgentError> {
```

Comment:
Is this a WIP? Otherwise why not have the `send` call directly in `ClientMessage::TcpSteal` handling?